.github/workflows/dbdocs.yml

```diff
@@ -28,7 +28,9 @@ jobs:
         with:
           node-version: ${{ env.NODE_VERSION }}
       - name: Install dbdocs
-        run: npm install dbdocs
+        run: sudo npm install -g dbdocs
```

Comment:
`sudo` is normally discouraged when using `npm` (for the same reason you never `sudo pip install ...`). does dbdocs require this?

Reply:
just saw that they say it "may be required". can we try without? it's a CI runner so maybe not too much impact of `sudo install`ing it, but I'd rather not if we don't have to

Reply:
It was still successful after removing `sudo`. I also updated the label. Good to merge?